lib/node/runner/api/blocks.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"net/http"
+	"strconv"
+
+	"boscoin.io/sebak/lib/block"
+	"boscoin.io/sebak/lib/common/observer"
+	"boscoin.io/sebak/lib/network/httputils"
+	"boscoin.io/sebak/lib/node/runner/api/resource"
+	"boscoin.io/sebak/lib/storage"
+)
+
+func (api NetworkHandlerAPI) GetBlocksHandler(w http.ResponseWriter, r *http.Request) {
+	p, err := NewPageQuery(r)
+	if err != nil {
+		httputils.WriteJSONError(w, err)
+		return
+	}
+
+	var (
+		cursor []byte // cursor as height
+		blocks []resource.Resource
+	)
+
+	var option *storage.WalkOption
+	{
+		height, err := strconv.ParseUint(string(p.Cursor()), 10, 64)
+		if err != nil {
+			height = 1 // default cursor is height 1
```

Comment:
We already defined, `common.GenesisBlockHeight`

Reply:
I changed it :->